consensus/tree_hash/src/merkle_hasher.rs

```diff
@@ -368,7 +368,7 @@ mod test {
     fn context_size() {
         assert_eq!(
             mem::size_of::<HalfNode>(),
-            232,
+            216,
```

Comment:
```suggestion
            224,
```

Sorry, should be 224. The inner `Context` is 216 bytes but we also store an extra `usize`